sys-apps/etckeeper/etckeeper-1.18.6-r1.ebuild

```diff
@@ -55,6 +55,13 @@ src_install(){
 		exeinto /etc/cron.daily
 		newexe debian/cron.daily etckeeper
 	fi
+
+	# remove subdirectories of /var which cause QA warnings
+	# bug #513374
+	rmdir "${ED%/}"/var/cache/etckeeper >/dev/null ||
+						die "Cannot remove /var subfolders (1)"
+	rmdir "${ED%/}"/var/cache >/dev/null || die "Cannot remove /var subfolders (2)"
+	rmdir "${ED%/}"/var >/dev/null	     || die "Cannot remove /var subfolders (3)"
```

Comment:
I think you can ignore these two. Re-installing them (they already exist) is a no-op.

Reply:
Well, you can ignore `/var`, but it looks like the QA warning is complaining about `/var/cache`. That's probably a bug in the QA check, but oh well =P

Reply:
The QA script (`bin/install-qa-check.d/20runtime-directories` in the portage source) says creating `/var/cache` should be OK:

```bash
runtime_dir_check() {
	# It's ok create these directories, but not to install into them. #493154
	# TODO: We should add var/lib to this list.
	local x f=
	for x in var/cache var/lock var/run run ; do
```

Whether that actually works or not, you'll have to try...